tests/fast_tests/test_environments.py

```diff
@@ -0,0 +1,383 @@
+import unittest
+import os
+import numpy as np
+
+from flow.core.vehicles import Vehicles
+from flow.core.params import NetParams, EnvParams, SumoParams, InitialConfig
+from flow.controllers import RLController, IDMController
+from flow.core.rewards import desired_velocity
+
+from flow.scenarios import LoopScenario, CircleGenerator
+from flow.scenarios.loop.loop_scenario import ADDITIONAL_NET_PARAMS \
+    as LOOP_PARAMS
+
+from flow.envs import TestEnv, AccelEnv, LaneChangeAccelEnv, \
+    LaneChangeAccelPOEnv, WaveAttenuationEnv, WaveAttenuationPOEnv
+from flow.envs.loop.loop_accel import ADDITIONAL_ENV_PARAMS as ACCELENV_PARAMS
+from flow.envs.loop.lane_changing import ADDITIONAL_ENV_PARAMS as LCENV_PARAMS
+from flow.envs.loop.wave_attenuation import ADDITIONAL_ENV_PARAMS as WAV_PARAMS
+
+os.environ["TEST_FLAG"] = "True"
+
+
+class TestTestEnv(unittest.TestCase):
+
+    """Tests the TestEnv environment in flow/envs/test.py"""
```

Comment:
This isn't a comment, but the name of this class is surreal.

Reply:
hahaha